File: client/src/app/models/form-models/LoginForm.ts
export class LoginForm {

  constructor(private gmail: string,
              private password: string){ }

  getGmail(): string {
    return this.gmail;
  }

  getPassword(): string {
    return this.password;
  }

  setGmail(value: string) {
    this.gmail = value;
  }

  setPassword(value: string) {
    this.password = value;
  }

}

File: src/main/java/com/app/demo/models/LoginForm.java
package com.app.demo.models;

public class LoginForm {

    private String gmail;

    private String password;

    public String getGmail() {
        return gmail;
    }

    public void setGmail(String gmail) {
        this.gmail = gmail;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    @Override
    public String toString() {
        return "LoginForm{" +
                "gmail='" + gmail + '\'' +
                ", password='" + password + '\'' +
                '}';
    }

}
